test/operator/operator_test.py

```diff
@@ -643,7 +642,7 @@ def test_nonlinear_functional_operators():
 
     assert not C.is_linear
     assert almost_equal(C(x), A(x * 2.0))
-    
```

Comment:
Right. However, in this case, most things happen by me saving the file, which removes all the trailing whitespaces *which should not be there in the first place*. I'll be more angry about those and other avoidable PEP8 violations in the future then :angry: